todo.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script type="text/javascript">
+</script>
+</head>
+
+<body>
+<h1>Todo List</h1>
+    <div id="todoList"></div>
+<hr>
+    <input type="text" id="todoDescription" placeholder="Add a todo here"/>
+    <button id="addTodo">Add</button>
+    
+    <div>Remaining tasks:<a id="remainingCount">0</a></div>
+    <div>Completed tasks:<a id="completedCount">0</a></div>
+<script >
+
+var totalTasks = 0;
+
+function addToList(todoItem, todoDescription) {
+    var newTodo = document.createElement("div");
```

Comment:
The construction of the HTML is happening using DOM creation functions. This works well, but it's very hard for someone else to read because they would literally have to piece together how it will look structurally. This is why you typically prefer to use string HTML representations rather than creating them by hand. This of course becomes easier with jQuery.